content/blog/TinaCMS-Version-21.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: TinaCMS Version 2.1
+date: '2024-07-24T14:00:00.000Z'
+last_edited: '2024-07-24T14:00:00.000Z'
+author: Jack Pettit
+---
+
+We are excited to announce the release of `tinacms@2.1.0` and `@tinacms/cli@1.5.51`!
+
+### What's New:
+
+1. **Removed Deprecated `useTina` Hook**
+   * This hook, deprecated since the v1 release, has now been fully removed. Note that this change pertains only to the legacy version used internally and does not affect the widely used `useTina` hook.
+2. **Next.js Version Upgrade**
+   * We've upgraded the Next.js version in TinaCMS packages from 12 to 14, along with several other dependency updates to ensure compatibility and improved performance.
+3. **PNPM Support for `create-tina-app`**
+   * You can now use PNPM as an option when creating a Tina app, offering more flexibility and efficiency in your package management.
+4. **React 18.3.1 Integration**
+   * TinaCMS has been upgraded to use React 18.3.1, bringing you the latest features and improvements from React.
+5. **New `toolbarOverride` Option for Rich Text Fields**
```

Comment:
I think this should link to the docs for toolbarOverride

Reply:
yea good point

Reply:
✅